pkg/dyff/output_human.go

```diff
@@ -361,8 +361,8 @@ func (report *HumanReport) highlightByLine(from, to string) string {
 
 	} else {
 		report.writeTextBlocks(&buf, 0,
-			red(createStringWithPrefix("  - ", from)),
-			green(createStringWithPrefix("  + ", to)),
+			red("%s", createStringWithPrefix("  - ", from)),
```

Comment:
Good catch. I fixed things like that in other places, but that slipped obviously.